helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -555,7 +569,11 @@ private void subscribeDataChange(String path, NotificationContext.Type callbackT
         logger.debug(_manager.getInstanceName() + " subscribe data-change. path: " + path
             + ", listener: " + _listener);
       }
-      _zkClient.subscribeDataChanges(path, this);
+      boolean rt = _zkClient.subscribeDataChanges(path, this, callbackType != Type.INIT);
```

Comment:
Could have been better to use java style naming? A clearer variable name would be appreciated.

Reply:
changed to subStatus.